src/main/java/org/togetherjava/tjbot/commands/AbstractCommand.java

```diff
@@ -0,0 +1,53 @@
+package org.togetherjava.tjbot.commands;
+
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.requests.restaction.CommandCreateAction;
+
+/**
+ * The class all commands have to abstract.
+ * Look at {@link org.togetherjava.tjbot.commands.example.ExampleCommand} for an example.
+ * A second example will come for sub-commands.
+ */
+public abstract class AbstractCommand {
```

Comment:
This might be able to be an interface if we never need local state:
```suggestion
public interface Command
```

Reply:
> Before this gets merged to develop, we will look back and change to an interface if no features of abstract classes will be used.
[My last message](https://github.com/Together-Java/TJ-Bot/pull/30#issuecomment-915054339)